core/templatetags/element_by_counter.py

```diff
@@ -0,0 +1,9 @@
+from django import template
+register = template.Library()
+
+@register.simple_tag
+def element_by_counter(data, counter, divider):
+     index = int(counter / divider)
```

Comment:
Let's make sure we handle `divider = 0` well. If it's `0` we should return empty string, too, instead ending up with division by zero error.

Reply:
Done, thanks.